package.json

```diff
@@ -34,13 +34,10 @@
     "security-checks": "git secrets --scan && git log -p | scanrepo",
     "clean": "rimraf ./.build ./.artifact ./.serverless ./.webpack ./*.zip",
     "compile": "tsc",
-    "build:dev": "cross-env NODE_ENV=local serverless webpack",
-    "build:production": "source config/.env.production && cross-env NODE_ENV=production webpack -c ./scripts/webpack.config.build.js --env=production",
-    "build": "npm-run-all clean build:dev",
+    "build": "node_modules/typescript/bin/tsc --rootDir ./ --outDir .build --sourceMap false && npm run build:copy && rimraf .build/tests",
```

Comment:
Can't this be just `tsc`?

Reply:
Works locally fine. Testing it in the pipeline...

Reply:
Works fine in pipeline too. Change made.